pkg/printers/internalversion/printers.go

```diff
@@ -876,7 +886,16 @@ func printPod(pod *api.Pod, options printers.GenerateOptions) ([]metav1.TableRow
 		}
 		break
 	}
-	if !initializing {
+
+	hasInitialized := false
```

Comment:
Can you add a comment explaining the logic behind this `hasInitialized` flag and why its needed now? I didn't understand it.